app/(splash)/github-callback/page.tsx

```
@@ -1,35 +1,59 @@
-          />
-          <Typo
-            variant={"heading"}
-            size={"md"}
-            color={"tertiary"}
-            translate={{
-              token: "splash:githubCallback.subtitle",
-            }}
-          />
-        </div>
-      </div>
+const GenericState = (<div className={"fixed inset-0 z-[9999]"}>
```

ESLint

[error] Insert `⏎··`

(prettier/prettier)